app.js

```diff
@@ -101,9 +101,31 @@ function displayBookSection() {
   }
 }
 
+function displayNewBooksSection() {
+  const sections = document.querySelectorAll('section');
+  sections.forEach((section) => {
+    if (section.id === 'new-books') {
+      section.style.display = 'flex';
+    } else {
+      section.style.display = 'none';
+    }
+  });
+}
+
+function displayContactSection() {
+  const sections = document.querySelectorAll('section');
+  sections.forEach((section) => {
+    if (section.id === 'contact') {
+      section.style.display = 'flex';
+    } else {
+      section.style.display = 'none';
+    }
```

Comment:
- [ ] Please make sure upon page refresh or page load only the active section is displaying instead of all three sections.

Your Website:

<img width="1270" alt="Screenshot 2023-07-05 at 3 05 14 PM" src="https://github.com/mryeminthu/Awesome-books/assets/44798044/eab91c36-b3c7-4f11-8023-8f61201d8ce4">

Reply:
Hi @shella12 . Sorry to bother. You reviewed [my partner and I's](https://github.com/mryeminthu/Awesome-books/pull/5) code an hour ago.

My partner and I aren't entirely sure what you mean about the "active section" change, as we don't see it as part of the requirements for the project. Could you explain a little further about this requirement? We can make it so only one section appears instead of all three simple enough, We just don't know what you mean regarding the active section